.github/workflows/ispc-ci.yml

```diff
@@ -0,0 +1,212 @@
+name: ISPC CI workflow
```

Comment:
The name is not informative.
It belongs to ISPC repo, so `ISPC` in the name is not useful.
It's a GitHub Action, so it's `CI` anyway.
And it's workflow, so no need to say `workflow` :)

Right now it's Linux only, but I assume we will add Windows and macOS to the same pipeline, so probably we should avoid saying Linux.

Then probably just `Tests`?